apps/cookbook/src/app/examples/link-example/examples/trailing-icon.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-link-example-trailing-icon',
+  template: `<a class="trailing-icon-example" target="_blank" href="https://github.com/kirbydesign/designsystem">Trailing icon</a>`,
+  style: `@use '@kirbydesign/core/src/scss/utils';
+.trailing-icon-example
+{
+@include utils.trailing-icon("navigation")
+/* 
+@include utils.trailing-icon("https://cdn.jsdelivr.net/npm/open-iconic@1.1.1/svg/heart.svg")
+*/
+}`,
```

Comment:
Consider readability:
```suggestion
  style: `@use '@kirbydesign/core/src/scss/utils';

.trailing-icon-example {
  // Using a Kirby icon:
  @include utils.trailing-icon("navigation");
  
  // Using a URL to an svg:
  @include utils.trailing-icon("https://cdn.jsdelivr.net/npm/open-iconic@1.1.1/svg/heart.svg");
}`,
```